src/main/resources/static/account-setting-view.html

```diff
@@ -0,0 +1,29 @@
+<!DOCTYPE html>
+<html lang="en">
```

Comment:
it should not change color on hover
![image](https://user-images.githubusercontent.com/10707837/36749984-b790f7ae-1bfc-11e8-92d4-aecea9202215.png)